src/invidious/views/template.ecr

```diff
@@ -106,6 +106,7 @@
 
             <%= content %>
 
+            <div id="placeholder" style="padding: 6em;"></div>
```

Comment:
Please remove the placeholder and add the "padding" to the footer itself. It should be possible to change the footer's `margin` to a `padding` and add the placeholder's `padding` as `margin-top`.

While the footer is modified we should also change the `<div class="footer">` (which should have been a `<div id="footer">` to begin with) into `<footer>`.

Reply:
Without the placeholder element things quickly overlap even after transferring the css.

![placeholder](https://user-images.githubusercontent.com/70992037/113703035-d3be7780-96c9-11eb-9087-0766b0845198.png)

Reply:
Oh, I overlooked the `position: absolute` and thought the placeholder was between the content and the footer, sorry.

Please use flexbox instead of absolute positioning.